docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -119,26 +119,25 @@ include an attribute for their relationship to a reference person
 (:ref:`5 <census_prl_age_sex_etc>`). Due to the complex interplay of
 these attributes we will need an enhanced fertility model (:ref:`6
 <census_prl_fertility>`).  We can use our standard mortality model
-(:ref:`7 <census_prl_mortality>`), but will need a totally new model
-of migration (:ref:`8 <census_prl_migration>`) that accounts for moves
-by household and individual simulants and allows migration in to and
-out of the tracked population, as well as changes to geographic
+(:ref:`7 <census_prl_mortality>`), but will need totally new models
+of domestic (:ref:`8 <census_prl_domestic_migration>`) and international (:ref:`9 <census_prl_international_immigration>`; :ref:`10 <census_prl_international_emigration>`) migration that account for moves
+by household and individual simulants, as well as changes to geographic
 location and household id.
 
 On top of this, we will layer attributes relevant to PRL: mailing
-addresses for each household (9); first, middle, and last names for
-each simulant (10); date of birth (11); intended-to-be-unique
+addresses for each household (11); first, middle, and last names for
+each simulant (12); date of birth (13); intended-to-be-unique
 identification number modeling SSN that is missing for some and not
-actually unique for others (12); and periodic survey, census, and registry
-observations with realistic noise (13).
+actually unique for others (14); and periodic survey, census, and registry
+observations with realistic noise (15).
 
 Additional components we might want: time-dependent changes to
-observers of sex, based on gender assigned at birth (15); multiple
-households for individuals, leading to double counting in census (16);
-twins and multiparous births in fertility model (14).  To capture an
+observers of sex, based on gender assigned at birth (17); multiple
+households for individuals, leading to double counting in census (18);
```

Comment:
We want an observer that double counts individuals?

Reply:
Yes, @SylLutze is working on documenting this. Here I've just changed the number of the component.

Reply:
Even triple counts sometimes!

Reply:
@zmbc I think we should connect on double counting. I think ideally the double (or triple) counting happening in the simulation, not in the observer. 

An example: someone should move and then exist in the simulation in two households. Rather than at the point of sampling point having them be double counted.

Reply:
While I think we will stick with the simpler approach where they are double-counted at enumeration time, it would be interesting to think through _where_ they are double-counted. I'd imagine a good amount of double-counting (>50%?) happens with the duplicates in _different_ households, so double-counting them with the same household ID would make PRL look too easy.

Reply:
Agreed. It sounds like for now that we will basically not be including this for now in the observer in any function - same household ID or not. I will list that in the limitations but I think its important to include somehow and have some folks be listed at multiple addresses and some duplicated at the same address.